libs/martech/attributes.js

```diff
@@ -11,6 +13,13 @@ export function processTrackingLabels(text, config, charLimit) {
   return analyticsValue;
 }
 
+function getHeaderCharLimit(str) {
+  const defaultLimit = 20;
+  if (str === null || str === undefined) return defaultLimit;
+  if (str === 'off') return 0;
+  if (!Number.isNaN(Number(str))) return Number(str);
```

Comment:
```suggestion
  if (!isNaN(str)) return parseInt(str);
```

Reply:
I kept the Number.isNan in the condition because it throws a linting error without it and I've updated to use parseInt(str, 10) to insure integer value. Thanks.